fortitude/src/check.rs

```diff
@@ -197,14 +198,21 @@ pub fn check(args: CheckArgs) -> Result<ExitCode> {
                         total_errors += 1;
                     }
                 }
+                total_files += 1;
             }
+            let file_no = format!(
+                "fortitude: {} files scanned.",
+                total_files.to_string().bold()
+            );
             if total_errors == 0 {
+                let success = "All checks passed!".bright_green();
+                println!("\n{}\n{}\n", file_no, success);
                 Ok(ExitCode::SUCCESS)
             } else {
                 let err_no = format!("Number of errors: {}", total_errors.to_string().bold());
                 let info = "For more information, run:";
                 let explain = format!("{} {}", "fortitude explain", "[ERROR_CODES]".bold());
-                println!("\n-- {}\n-- {}\n\n    {}\n", err_no, info, explain);
+                println!("\n{}\n{}\n\n{}\n\n    {}\n", file_no, err_no, info, explain);
```

Comment:
If we're touching this, may as well use the f-string style syntax?

```suggestion
                println!("\n{file_no}\n{err_no}\n\n{info}\n\n    {explain}\n");
```